docs-source/content/userguide/managing-operators/using-the-operator/using-helm.md

```diff
@@ -77,6 +77,8 @@ $ helm upgrade \
   kubernetes/charts/weblogic-operator
 ```
 
+Refer to [Domain Namespace Management] ({{<relref "/faq/namespace-management.md">}}) for more information about managing `domainNamespaces`.
```

Comment:
Please add a new section to 'common mistakes and solutions' titled something like 'Problems when removing or recreating a namespace'.   And include text something like "Problems can occur when forgetting to inform the operator about a removed namespace, or forgetting to add a namespace back when a namespace is deleted and recreated."   Then add reference to the FAQ.

Reply:
I am holding on making changes in the cross-references areas until we decide if we want to keep the content in this PR as a FAQ entry or move it into the user guide using-helm.md.

Reply:
Modified the cross-references.